cmd/larry/main.go

```diff
@@ -33,6 +36,13 @@ var (
 	twitterConsumerSecret = envString("TWITTER_CONSUMER_SECRET", "")
 	twitterAccessToken    = envString("TWITTER_ACCESS_TOKEN", "")
 	twitterAccessSecret   = envString("TWITTER_ACCESS_SECRET", "")
+
+	mastodonClientId     = envString("MASTODON_CLIENT_ID", "")
+	mastodonClientSecret = envString("MASTODON_CLIENT_SECRET", "")
+	mastodonServer       = envString("MASTODON_SERVER", "")
+	mastodonAccessToken  = envString("MASTODON_ACCESS_TOKEN", "")
+	mastodonAccount      = envString("MASTODON_ACCOUNT", "")
+	mastodonPassword     = envString("MASTODON_PASSWORD", "")
```

Comment:
I would probably move all this into a `cli.Flag` to utilize the cli package "better". But I stopped myself. Can send a follow-up if you don't mind.

Reply:
Yeah, I added them there for readability, but I can create a new issue to change this